default/scripting/specials/planet/TEMPORAL_ANOMOLY.focs.txt

```diff
@@ -15,12 +15,12 @@ Special
                 Source
                 Focus type = "FOCUS_RESEARCH"
             ]
-            priority = [[EARLY_PRIORITY]]
+            priority = [[EARLY_TARGET_POPULATION_POST_SCALING_PRIORITY]]
```

Comment:
Looks wrong.

Reply:
Yes.  I don't think this difference actually matters, but I did note it in the Issue thread, it was a mistaken change that should not remain.